sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -72,6 +75,21 @@
     return event;
   }
 
+  private void setDebugMeta(final @NotNull SentryEvent event) {
+    if (event.getDebugMeta() == null && !options.getProguardUuids().isEmpty()) {
+      final DebugMeta debugMeta = new DebugMeta();
+      final List<DebugImage> images = new ArrayList<>();
+      for (final String uuid : options.getProguardUuids()) {
+        final DebugImage debugImage = new DebugImage();
+        debugImage.setType("proguard");
```

Comment:
lets create a constant for the String, so we reuse it here https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java#L289

Reply:
👍